src/cloud/project/user-admin.md

```diff
@@ -104,42 +105,76 @@ magento-cloud user:role alice@example.com --level environment --environment deve
 {:.bs-callout-info}
 To list the available `magento-cloud` CLI commands, use the `magento-cloud list` command.
 
-### Manage users from the Project Web UI {#cloud-user-webinterface}
+## Manage users from the Project Web UI {#cloud-user-webinterface}
 
-{:.procedure}
-To create user accounts from the Project Web UI:
+You can add project-level and environment-level users from the Project Web UI, and use the _Edit_ feature to modify permissions for an existing user.
 
-1. Log in to [your {{site.data.var.ece}} account](https://accounts.magento.cloud).
+### Open the Project Web UI
+
+1. Log in to [your {{site.data.var.ece}} account][].
 
 1. Click the **Projects** tab as the following figure shows.
 
    ![Click the projects tab to access your Cloud project]({{ site.baseurl }}/common/images/cloud/cloud_account_project.png){:width="550px"}
 
-1. Click the name of your project.
+1. Click the name of your project to open the Cloud project portal (Onboarding UI).
+
+1. Click **Infrastructure access**, and then click the **Project Access (Web UI)** link.
+
+   ![Cloud project portal]({{ site.baseurl }}/common/images/cloud/cloud-login-infrastructure-access.png){:width="550px"}
+
+1. In the Project Web UI, add project-level users and environment-level users as needed.
+
+   When you add the user, the user receives an email inviting them to join the {{site.data.var.ece}} project with instructions for account registration and email verification.
+
+### Add a project-level user
 
-1. Click the configure project button next to project name in the top navigation bar as the following figure shows.
+1. Click the configure project gear icon next to project name in the top navigation bar.
```

Comment:
Step one should always include where to begin, in case the reader didn't see the previous procedure. "In the Project Web UI..." or something like that.